queries.sql

```diff
@@ -0,0 +1,31 @@
+/* Find all animals whose name ends in "mon". */
+
+SELECT * FROM animals WHERE name like '%mon';
+
+/* List the name of all animals born between 2016 and 2019. */
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-10';
```

Comment:
* Kindly adjust this query as animals born on 2019-12-31 might not get captured in this query, Kindly extend your search to the end of 2019 so that this way all animals born between 2016 and 2019 will be covered